plugins/slipping-away-users/frontend/public/javascripts/countly.models.js

```diff
@@ -1,30 +1,106 @@
-/*global countlyCommon,jQuery*/
-(function(countlySlippingPlugin, $) {
-    var _data = {};
-    /**
-     * This is for initializing model
-     * @param {object} query user drill filter query
-     * @return {func} ajax func to request data and store in _data
-    */
-    countlySlippingPlugin.initialize = function(query) {
+/*global countlyCommon,jQuery,countlyVue*/
+(function(countlySlippingAwayUsers, $) {
 
```

Comment:
Is there another plugin which uses this? If so, deleting the old API could break things.

Reply:
I was not able to find any plugin that does it (except for its former view beforeRender() method, which has become obsolete now).